apps/processor/README.md

```diff
@@ -0,0 +1,61 @@
+# Processor Service
+
+This service is the main application that runs the core processing pipeline:
+
+-   Instantiates and coordinates components from [Grants Stack Indexer packages](../../packages/)
+-   Creates and manages an Orchestrator per chain to process blockchain events
+
+## Requirements
+
+-   A running instance of PostgreSQL Data Layer with migrations applied
+-   A running instance of Envio Indexer
+
+## Setup
+
+1. Install dependencies running `pnpm install`
+2. Build the app using `pnpm build`
+
+### ⚙️ Setting up env variables
+
+-   Create `.env` file and copy paste `.env.example` content in there.
+
+```
+$ cp .env.example .env
+```
+
+Available options:
+| Name | Description | Default | Required | Notes |
+|-----------------------------|--------------------------------------------------------------------------------------------------------------------------------|-----------|----------------------------------|-----------------------------------------------------------------|
+| `RPC_URLS` | Array of RPC URLs | N/A | Yes | Multiple URLs for redundancy |
+| `CHAIN_ID` | Chain ID | N/A | Yes | At the moment only Optimism is supported (10) |
+| `FETCH_LIMIT` | Maximum number of items to fetch in one batch | 500 | Yes | |
+| `FETCH_DELAY_MS` | Delay between fetch operations in milliseconds | 3000 | Yes | |
+| `DATABASE_URL` | PostgreSQL Data Layer database connection URL | N/A | Yes | |
+| `DATABASE_SCHEMA` | PostgreSQL Data Layer database schema name | chainDataSchema | Yes | |
+| `INDEXER_GRAPHQL_URL` | GraphQL endpoint for the indexer | N/A | Yes | |
+| `INDEXER_ADMIN_SECRET` | Admin secret for indexer authentication | N/A | Yes | |
```

Comment:
we should change this, admin secret is just for admins, should be a way to generate some token for client auth

Reply:
https://linear.app/defi-wonderland/issue/GIT-127/replace-admin-secret-with-token-for-indexer-clients